models/category.ts

```diff
@@ -0,0 +1,10 @@
+const { Pool } = require('pg');
+
+const pool = new Pool();
+const Logger = require('../models/logger');
+
+
+module.exports.getCategories = function (callback: Function) {
```

Comment:
Would work but this isnt the correct naming for Postgresql. The correct table name would be "categories". But this works anyhow